### PR TITLE
add lambda chain registry infos

### PR DESCRIPTION
**Pull request by steveliupai**

### CODE REVIEW

**Comment by HeesungB** [comment]
@steveliupai Hello. Could you add node provider information?

```
Node provider should be provided
```

**Comment by steveliupai** [comment]
> @steveliupai Hello. Could you add node provider information?
> 
> ```
> Node provider should be provided
> ```

@HeesungB Hi HeesungB, just added node provider infos, please approve to merge. Thank you so much!

**Comment by HeesungB** [comment]
@steveliupai Could you resize image to 256x256px?

```
Image size is not 256x256px. size : {"height":497,"width":497,"type":"png"}
```

**Comment by steveliupai** [comment]
> @steveliupai Could you resize image to 256x256px?
> 
> ```
> Image size is not 256x256px. size : {"height":497,"width":497,"type":"png"}
> ```

@HeesungB Thank you so much for pointing out! Just replace with 256x256px. size icon

File: cosmos/lambda_92000.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "website": "https://nodestake.top"
+  },
+  "stakeCurrency": {
+    "coinDenom": "LAMB",
+    "coinMinimalDenom": "ulamb",
+    "coinDecimals": 18,
+    "coinGeckoId": "lambda",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/lambda_92000/lambda.png"
+  },
+  "walletUrl": "https://lambda.im/",
```
Comment (HeesungB) [inline cosmos/lambda_92000.json:23]: @steveliupai Could you remove `walletUrl` field?

Comment (steveliupai) [reply cosmos/lambda_92000.json]: done

Comment (steveliupai) [reply cosmos/lambda_92000.json]: @HeesungB Anything else we need to fix?

**Comment by HeesungB** [comment]
@steveliupai Thank you for contribution!

**Pull request merged**